data/setup.xml

```diff
@@ -151,8 +151,8 @@
 			<item level="1" text="Show PiP in the Front Display" description="You can Display PiP with or without OSD Menu.">config.lcd.modepip</item>
 			<item level="1" text="Set Framerate for MiniTV" description="Lower Framerate saves CPU Time.">config.lcd.fpsminitv</item>
 		</if>
-		<item level="1" text="REC Symbol in use" description="Enable blinking rec symbol on the LCD Display.">config.usage.blinking_rec_symbol_during_recording</item>
-		<item level="1" text="REC Symbol in Standby" description="Enable blinking rec symbol on the LCD Display in Standby.">config.usage.blinking_display_clock_during_recording</item>
+		<item level="1" text="REC Symbol in use" description="Enable blinking rec symbol on the LCD Display." requires="LCDSKINSetup">config.usage.blinking_rec_symbol_during_recording</item>
```

Comment:
Are you sure about the "requires"?

Reply:
I think blinking is set in a display skin and these are related to
BoxInfo.setItem("LCDSKINSetup", fileExists("/usr/share/enigma2/display")).

Reply:
No `fileExists("/usr/share/enigma2/display")` No blinking